### PR TITLE
fix(docker): move comment up

### PR DESCRIPTION
**Pull request by cbernat**
Inline comment caused script not working.

### CODE REVIEW

File: watch-w-docker.sh [hunk 1]
```diff
@@ -10,11 +10,12 @@ cd $(dirname $0)
 export MSYS_NO_PATHCONV=1
 export MSYS2_ARG_CONV_EXCL="*"
 
+# -e CHOKIDAR_USEPOLLING=true \ --> hot reload, beware of CPU usage
 docker run --rm \
     -p 3000:3000 \
     -v `pwd`:/work \
     -w /work \
-    -e CHOKIDAR_USEPOLLING=true \ #hot reload, beware of CPU usage
+    -e CHOKIDAR_USEPOLLING=true \
```
Comment (andresmoschini) [inline watch-w-docker.sh]: Ohh, I did not see it

Comment (cbernat) [reply watch-w-docker.sh]: It was my mistake for not running it first :/ Now I've tested and it works.

**Comment by cbernat** [comment]
Test this please

**Comment by cbernat** [comment]
Test this please

**Pull request merged**